src/components/molecules/NavBar/NavBar.scss

```diff
@@ -487,7 +487,7 @@ $border-radius: 28px;
     &:nth-child(odd) {
       background-color: rgba($black, 0.3);
     }
-    &.schedule-event-container_live {
+    &.schedule-event-container--live {
```

Comment:
This is legacy style logic and I didn't want to mess with it

Reply:
Makes sense